### PR TITLE
Update onboarding

### PR DESCRIPTION
**Pull request by astrojuanlu**
Addresses #8410 by

1. Directing users to read the Read the Docs tutorial https://docs.readthedocs.io/page/tutorial/ rather than the Getting Started with Sphinx guide
2. Replaces the demo (which was a dead end) by the same template used in the tutorial

This is how it looks like now:

Screenshot 2021-09-16 at 15-26-46 Project Dashboard Read the Docs

⚠️ Notice that https://github.com/readthedocs/tutorial-template/ doesn't exist yet, until we agree to transfer https://github.com/astrojuanlu/tutorial-template ⚠️

This also removes the demo code.

### CODE REVIEW

**Review by humitos — APPROVED** [review]
This is much better UX! 💯 

You may need to do a grep in other repositories (in particular -corporate) because there may be views that still depend on this class. Also, you will need to double check that the `onboard_import.html` file on -corporate does not require a change.

That said, I think we can't merge this PR without checking those things.

**Comment by humitos** [comment]
> warning Notice that [readthedocs/tutorial-template](https://github.com/readthedocs/tutorial-template/) doesn't exist yet, until we agree to transfer [astrojuanlu/tutorial-template](https://github.com/astrojuanlu/tutorial-template) warning

+1 on transfering this

**Comment by astrojuanlu** [comment]
Good point, will check the corporate site now.

By the way, I don't understand the CircleCI failure 🤔 https://app.circleci.com/pipelines/github/readthedocs/readthedocs.org/2681/workflows/739e8fbc-0291-4bae-b3e9-8a308e633c54/jobs/5268?invite=true#step-105-354

**Review by ericholscher — APPROVED** [review]
Yay deleting code :)

File: readthedocs/templates/projects/onboard_import.html [hunk 1]
```diff
@@ -4,20 +4,21 @@
 <div class="onboard onboard-import-project">
   <h2>{% trans "Ready to share your documentation" %}{% if request.user.first_name %}, {{ request.user.first_name }}{% endif %}?</h2>
 
-  {% with getting_started_url="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html" %}
+  {% with tutorial_url="https://docs.readthedocs.io/page/tutorial/" template_project_url="https://github.com/readthedocs/tutorial-template/" %}
```
[the remark this answers is not in the document]
Comment (astrojuanlu) [reply readthedocs/templates/projects/onboard_import.html:7]: Done ✔️

File: docs/tutorial/index.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ Preparing your project on GitHub
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To start, `sign in to GitHub <https://github.com/login>`_
-and navigate to `the tutorial GitHub template <https://github.com/astrojuanlu/tutorial-template/>`_,
+and navigate to `the tutorial GitHub template <https://github.com/readthedocs/tutorial-template/>`_,
```
[the remark this answers is not in the document]
Comment (astrojuanlu) [reply docs/tutorial/index.rst:27]: Done! 🚀

**Comment by astrojuanlu** [comment]
Migrated the tutorial template to our GitHub organization https://github.com/readthedocs/tutorial-template/ and replaced another two links on the homepage to point to the Tutorial. Merging!

**Pull request merged**